core/src/main/java/de/danoeh/antennapod/core/storage/FeedSearcher.java

```diff
@@ -77,6 +78,7 @@ private FeedSearcher(){}
             e.printStackTrace();
         }
         Collections.sort(result, new SearchResultValueComparator());
+        Collections.sort(result, new InReverseChronologicalOrder ());
```

Comment:
Why does it need to be sorted two times? Please also remove the space between Order and (